services/storage/src/simcore_service_storage/models.py

```diff
@@ -168,9 +174,19 @@ def convert_from_lower_case(cls, v):
             return f"{v}".upper()
         return v
 
+    @root_validator()
+    @classmethod
+    def when_directory_force_link_type_and_file_size(cls, values):
+        if values["is_directory"] is True:
+            # sets directory size by default to undefined
+            values["file_size"] = -1
```

Comment:
MINOR: use constant to express what -1 means. e.g. `UNDEFINED_SIZE` (besides the doc) or  `FOLDER_WITH_NO_SIZE`